src/main/java/game/TicTacToe.java

```diff
@@ -1,29 +1,52 @@
 package game;
 
-import input.InputReader;
-import output.message.MessagePrinter;
-import output.message.Messages;
+import coordinates.Coordinates;
 import player.Player;
+import table.Table;
+import user.input.InputReader;
+import user.output.message.MessagePrinter;
+import user.output.message.Messages;
 
-public class TicTacToe {
+import java.util.Scanner;
+import java.util.function.Supplier;
+
+public final class TicTacToe {
   
-  Player nextPlayer = Player.X;
+  private Player nextPlayer = Player.X;
+  private final Scanner scanner = new Scanner(System.in);
+  private final Supplier<String> supplier = scanner::nextLine;
   
   public void letsPlay() {
     
     MessagePrinter.printMessage(Messages.WELCOME);
-    
-    boolean endOfTheGame = false;
-    while (!endOfTheGame) {
+    final int tableSizeFromUser = readLineToInt();
+  
+    boolean andTheWinnerIs = false;
```

Comment:
zmień tą nazwę, końcówka sugeruje, że to zwróci gracza, nie wartość prawda/fałsz
poza tym może niech to będzie licznik rozegranych partii?